apps/champions/priv/skills.json

```diff
@@ -1,12 +1,1651 @@
+    "effects": [
+      {
+        "type": {
+   "type": "instant"
+},
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "random",
+        "target_count": 1,
+        "target_allies": false,
+        "target_attribute": "Health"
+      }
+    ],
+    "cooldown": 5      
+  },
+  {
+    "name": "Qurin Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+   "type": "instant"
+},
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1.8,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "random",
+        "target_count": 2,
+        "target_allies": false,
+        "target_attribute": "Health"
+      }
+    ]
+  },
+  {
+    "name": "Chekla Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+   "type": "instant"
+},
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "random",
+        "target_count": 1,
+        "target_allies": false,
+        "target_attribute": "Health"
+      }
+    ],
+    "cooldown": 5      
+  },
+  {
+    "name": "Chekla Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+   "type": "instant"
+},
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [],
+        "executions": [
+          {
+            "type": "Heal",
+            "attack_ratio": 1.8,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "min",
+        "target_count": 1,
+        "target_allies": true,
+        "target_attribute": "Health"
+      }
+    ]
+  },
+  {
+    "name": "Germund Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+   "type": "instant"
+},
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "nearest",
+        "target_count": 1,
+        "target_allies": false,
+        "target_attribute": "Health"
```

Comment:
target_attribute like this one are wrong. "Nearest" strategy should not take a `target_attribute`. Only Min and Max.

Reply:
Also, isn't target_attribute as a whole outdated? We already have this
```
  def cast(%{"lowest" => attribute}), do: {:ok, {"lowest", attribute}}
  def cast(%{"highest" => attribute}), do: {:ok, {"highest", attribute}}

in our TargetStratgy type
```